app/src/main/res/values/strings.xml

```diff
@@ -359,6 +360,27 @@
     <string name="local_bundle_description">Import local files from your storage, does not automatically update</string>
     <string name="remote_bundle_description">Import remote files from a URL, can automatically update</string>
     <string name="recommended">Recommended</string>
+
+    <string name="installation_failed_dialog_title">Installation failed</string>
+    <string name="installation_cancelled_dialog_title">Installation cancelled</string>
+    <string name="installation_blocked_dialog_title">Installation blocked</string>
+    <string name="installation_conflict_dialog_title">Installation conflict</string>
+    <string name="installation_incompatible_dialog_title">Installation incompatible</string>
+    <string name="installation_invalid_dialog_title">Installation invalid</string>
+    <string name="installation_storage_issue_dialog_title">Not enough storage</string>
+    <string name="installation_timeout_dialog_title">Installation timed out</string>
+    <string name="installation_success_dialog_title">Installation succeeded</string>
+    <string name="installation_failed_description">The installation failed due to an unknown reason. Try again?</string>
```

Comment:
```suggestion
    <string name="installation_failed_description">The installation failed due to an unknown reason. Try again?</string>
```

Because of https://github.com/ReVanced/revanced-manager/pull/1473#issuecomment-2254259701